README.md

```diff
@@ -193,6 +193,42 @@ dbt-invoke properties.delete <options>
   - `<options>` uses the same arguments as for creating/updating property
     files, except for `--threads`.
 
+
+### Applying default properties using `dbt_invoke_template.yml`
+
+You can pass custom default properties to be added to property files on update.
+To do so, add a `dbt_invoke_template.yml` at the root of your dbt project,
+using the following convention:
+
+
+```
+model:
+  meta:
+    owner: "@default"
```

Comment:
This is an honest question from me (I don't have experience using the meta property):

Is there an advantage to doing this with dbt-invoke as opposed to setting a default meta config directly in dbt_project.yml (https://docs.getdbt.com/reference/model-configs#general-configurations)?

Sorry if I'm confusing different concepts.

Reply:
Using `dbt_project.yml` might sometimes be better, I believe, for dbt projects that are more mature and more stable. 

Ours is still young; we create a lot of models and move things around, which makes it harder for us to maintain ownership using this approach. Having the information at the model level is very convenient: we can just move both `.sql` and `.yml` files at once when re-structuring a folder, for example.

Else, we'd need to get back to `dbt_project.yml`, and edit the yml structure to reflect the new file layout. 

We tend to define configs using `dbt_project.yml` for things that do not move that much, eg. materializing views for our `/staging` folder.

Let me know if it doesn't make sense!